src/converter/dimacs/Syntax/LogExpSyntax.ts

```diff
@@ -0,0 +1,30 @@
+import { Rule } from "@jlguenego/lexer";
+import { and, negate, negatedVariable, or, variableRegexPart } from "./CommonSyntax";
+
+// export const negateRegexPart = '(?:\\b(?:not|NOT)\\s+|(?:\\b!\\s*))';
+export const negateRegexPart = '(?:(?:not|NOT)\\s+|(?:!\\s*))';
+
+export const regexAND = /(?:&|and|AND) /;
+export const regexOR = /(?:\||or|OR) /;
```

Comment:
I can add if you think that should be the case. I didn't add it initially because I thought it might be confusing to use */+ in both as a prefix or function-style way in dimacs and as binary operator in logical expressions.